.github/workflows/ci.yml

```diff
@@ -101,11 +109,12 @@ jobs:
           CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
           ASYNC_STD_THREAD_COUNT: 4
 
-      - name: Run tests shared-memory
+      - name: Run tests with SHM
+        if: ${{ matrix.os == 'ubuntu-latest' }}
         uses: actions-rs/cargo@v1
         with:
-          command: test
-          args: --features "transport_tcp,shared-memory" -p zenoh-transport --test unicast_shm
+          command: nextest
+          args: run -F shared-memory -F transport_shm
```

Comment:
This would rerun the full test suite, which will take way too long. The reason of having only `unicast_shm` test is to reduce testing time.